lib/request.js

```diff
@@ -91,6 +97,11 @@ module.exports = class Request {
       timestamp: Date.now() + this._timeOffset
     };
 
+    // Optional, set to 5000 by default. Increase if timestamp/recv_window errors are seen.
+    if(this.options.recvWindow) {
```

Comment:
I've made the change, though to reduce risk of typos I've ensured consistency across these params. It's always `recv_window` now. The new sync interval ms option is also similarly written `sync_interval_ms`, instead of camel case.